op-batcher/batcher/channel.go

```diff
@@ -226,23 +214,23 @@ func (s *channel) OutputFrames() error {
 }
 
 // LatestL1Origin returns the latest L1 block origin from all the L2 blocks that have been added to the channel
-func (c *channel) LatestL1Origin() eth.BlockID {
-	return c.channelBuilder.LatestL1Origin()
+func (s *channel) LatestL1Origin() eth.BlockID {
```

Comment:
Curious if you think resolving the receiver variable the other way would make more sense? i.e. using `(c *channel)` everywhere?

Reply:
@geoknee happy to, I did it this way to keep the diff small. Could I do that in a follow up, so we don't mix functionality changes with a variable rename?